admin/tabs/general/roles/details/members/RoleMembersModel.ts

```diff
@@ -47,13 +47,10 @@ export class RoleMembersModel extends HoistModel {
     }
 
     @computed
-    get directCounts(): Record<RoleMemberType, number> {
+    get inheritedRolesCount(): Record<string, number> {
```

Comment:
I see why you're doing this for symmetry, but to me it would be clearer to just return a number here.